bin/kube-divergence-simulation

```diff
@@ -178,22 +181,23 @@ kubectl get pods
 echo "All tasks are finished."
 
 # Download data from Redis at the end of the execution.
-if [ "$GCE" ]; then
+if [ "$GCE" == "true" ]; then
   echo "Running in Google Container Engine."
   export REDIS_SERVICE_HOST=$(kubectl get services | grep redis | awk '{print $3}' | head -1)
   # export REDIS_SERVICE_HOST=$(gcloud compute forwarding-rules list | grep TCP | awk '{print $3}' | tail -1)
   export REDIS_SERVICE_PORT=6379
-  echo "REDIS_SERVICE_HOST: ${REDIS_SERVICE_HOST}"
-  echo "REDIS_SERVICE_PORT: ${REDIS_SERVICE_PORT}"
 else
   export REDIS_SERVICE_HOST=$(kubectl config view | grep server | cut -f 2- -d ":" | tr -d " " | head -1 | cut -f 2-2 -d ":" | sed -e 's/\/\///g')
   export REDIS_SERVICE_PORT=$(kubectl get service redis -o jsonpath="{.spec.ports[*].nodePort}")
 fi
 
+echo "REDIS_SERVICE_HOST: ${REDIS_SERVICE_HOST}"
+echo "REDIS_SERVICE_PORT: ${REDIS_SERVICE_PORT}"
+
 echo "Downloading data from Redis."
 ./sync-redis.erl
-mkdir -p ~/Documents/lasp/priv/evaluation/logs
-cp -Rp /tmp/logs/* ~/Documents/lasp/priv/evaluation/logs
+mkdir -p ~/lasp/priv/evaluation/logs
```

Comment:
Can we parameterize this?  We both have different paths to where the lasp source code is.